Examples/MAX32665/OTP_Dump/project.mk

```diff
@@ -0,0 +1,9 @@
+# This file can be used for project configuration.
+# It's a sibling to the core "Makefile", which offers
+# various configuration variables that you can set here
+# if the default setup isn't suitable.
+
+# See the comments in the "Makefile" for a detailed
+# description of the default behavior and the full list of
+# available options.
+
```

Comment:
Please add below lines too
See: https://github.com/Analog-Devices-MSDK/msdk/blob/main/Examples/MAX32665/Hello_World/project.mk

# If you have secure version of MCU (MAX32666), set SBT=1 to generate signed binary
# For more information on how sign process works, see
# https://www.analog.com/en/education/education-library/videos/6313214207112.html
SBT=0